Switch back to dynamic include_role in logging loops

sdodson opened the pull request:
We'd switched to import_role to avoid increased memory consumption but
we must use include_role whenever we loop.

sdodson commented:
We probably need to find some way to ensure that import_role is not use with loops elsewhere. This is what what I found via a quick manual sweep of the logging role.
CC @richm

michaelgugino commented:
I think it's generally bad practice to loop over roles.  It's not obvious in the role itself that something is looping over it.

The impact here should be minimal as the actions are only executed against first_master, I believe.

richm commented:
/test install
because `Required Github CI test is not green: ci/openshift-jenkins/install`

sdodson commented:
/test install

richm commented:
/cherrypick release-3.7

richm commented:
@sdodson @jcantrill @michaelgugino this issue affects release-3.7 branch as well.  I tried to cherrypick this to 3.7 but got the above error.  Not sure what the problem is. This PR needs to be backported to 3.7.

sdodson commented:
> @sdodson @jcantrill @michaelgugino this issue affects release-3.7 branch as well. I tried to cherrypick this to 3.7 but got the above error. Not sure what the problem is. This PR needs to be backported to 3.7.
> 
>  Delete branch

We switched release-3.7 to static includes in a different manner because import_role doesn't exist in ansible 2.3 and we don't require ansible 2.4 in that release. That was done in #6613 there's only one change there related to logging which I've reverted in #6724
